content/en/docs/tutorials/kubernetes-basics/explore/explore-intro.html

```diff
@@ -153,8 +153,13 @@ <h3>Show the app in the terminal</h3>
                 To get the Pod name and store it in the <tt>POD_NAME</tt> environment variable:</p>
                 <p><code><b>export POD_NAME="$(kubectl get pods -o go-template --template '{{range .items}}{{.metadata.name}}{{"\n"}}{{end}}')"</b></code><br />
                 <code><b>echo Name of the Pod: $POD_NAME</b></code></p>
+                <p>In Windows system could use:</p>
```

Comment:
It's recommended to use bullets to describe different cases like:

- macOS
- Windows
- Other